src/cdk-experimental/popover-edit/edit-events.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Subject, timer} from 'rxjs';
+import {audit, distinctUntilChanged, filter, map, share} from 'rxjs/operators';
+import {Injectable} from '@angular/core';
+import {closest, CELL_SELECTOR, DEFAULT_HOVER_DELAY_MS, ROW_SELECTOR} from './common';
+
+/**
+ * Service for sharing delegated events and state for triggering edits.
+ */
+@Injectable()
+export class EditEvents {
+  readonly editing = new Subject<Element|null>();
+  readonly hovering = new Subject<Element|null>();
+  readonly mouseMove = new Subject<Element|null>();
+
+  protected currentlyEditing: Element|null = null;
+
+  protected readonly hoveringDistinct = this.hovering.pipe(distinctUntilChanged(), share());
+  protected readonly editingDistinct = this.editing.pipe(distinctUntilChanged(), share());
+
+  constructor() {
+    this.editing.subscribe(cell => {
```

Comment:
Can we just set this when `editingCell` is called rather than using a subscription?

Reply:
This happens only once and editingCell is called for every editable cell in the table.